### PR TITLE
Add patchstep "MERGE_CONTENT" (...and add 'v' to mod option version display)

### PR DESCRIPTION
**Pull request by EL20202**
MERGE_CONTENT would simply allow one to bulk add/modify many keys or array elements all at once.

To give an example, with an initial state of:
```json
{
    "a": 1,
    "b": 2
}
```
and a patchstep of:
```json
{
    "type": "MERGE_CONTENT",
    "content": {
        "a": 3,
        "c": 4
    }
}
```
It would modify the state to be:
```json
{
    "a": 3,
    "b": 2,
    "c": 4
}
```

For an array, it would work the same way - effectively acting as multiple `ADD_ARRAY_ELEMENT` to the end of the array. Functionally, it is identical to using `IMPORT` on a file holding the contents - removing the need to create a second file holding data to patch in, and also removing the need for multiple `SET_KEY` and `ADD_ARRAY_ELEMENT` patchsteps.

&nbsp;

As for the version display, it simply just adds a 'v' to the beginning of the version string. Probably not the more important part of this PR, but it seemed like a waste to do a whole PR over _just_ that.
image

### CODE REVIEW

**Review by 2767mr — APPROVED** [review]
Once a patch-steps maintainer gives approval I'll merge it

**Review by ac2pic — APPROVED** [review]

**Pull request merged**
